app/waf/src/application.ts

```diff
@@ -51,5 +56,61 @@ export class WafApplication extends BootMixin(
         nested: true,
       },
     };
+
+    if (process.env.TEST_OPENSTACK_INTEGRATION)
```

Comment:
Can you briefly describe why we just behavior like this only for "TEST_OPENSTACK_INTEGRATION"?

Reply:
In product releaese, we don't need to expose openstack operation API as demo shows. 
So this is only for demo. Will remove it.